src/pages/developers/local-environment.tsx

```diff
@@ -0,0 +1,314 @@
+      background: "#ffffff",
+      name: "Brownie",
+      description: "page-developers-local-environment:page-local-environment-brownie-desc",
+      alt: "page-developers-local-environment:page-local-environment-brownie-logo-alt",
+      image: EthDiamondBlackImage,
+    },
+    {
+      id: "epirus",
+      url: "https://www.web3labs.com/epirus",
+      githubUrl: "https://github.com/web3labs/epirus-free",
+      background: "#ffffff",
+      name: "Epirus",
+      description: "page-developers-local-environment:page-local-environment-epirus-desc",
+      alt: "page-developers-local-environment:page-local-environment-epirus-logo-alt",
+      image: EpirusImage,
+    },
+    {
+      id: "createethapp",
+      url: "https://github.com/PaulRBerg/create-eth-app",
+      githubUrl: "https://github.com/PaulRBerg/create-eth-app",
+      background: "#ffffff",
+      name: "Create Eth App",
+      description: "page-developers-local-environment:page-local-environment-eth-app-desc",
+      alt: "page-developers-local-environment:page-local-environment-eth-app-logo-alt",
+      image: EthDiamondBlackImage,
+    },
+    {
+      id: "scaffoldeth",
+      url: "https://github.com/austintgriffith/scaffold-eth",
+      githubUrl: "https://github.com/austintgriffith/scaffold-eth",
+      background: "#ffffff",
+      name: "scaffold-eth",
+      description: "page-developers-local-environment:page-local-environment-scaffold-eth-desc",
+      alt: "page-developers-local-environment:page-local-environment-scaffold-eth-logo-alt",
+      image: ScaffoldEthImage,
+    },
+    {
+      id: "soliditytemplate",
+      url: "https://github.com/paulrberg/solidity-template",
+      githubUrl: "https://github.com/paulrberg/solidity-template",
+      background: "#ffffff",
+      name: "Solidity template",
+      description: "page-developers-local-environment:page-local-environment-solidity-template-desc",
+      alt: "page-developers-local-environment:page-local-environment-solidity-template-logo-alt",
+      image: EthDiamondBlackImage,
+    },
+    {
+      id: "foundry",
+      url: "https://getfoundry.sh/",
+      githubUrl: "https://github.com/foundry-rs/foundry",
+      background: "#ffffff",
+      name: "Foundry",
+      description: "page-developers-local-environment:page-local-environment-foundry-desc",
+      alt: "page-developers-local-environment:page-local-environment-foundry-logo-alt",
+      image: FoundryImage,
+    },
+  ]
+  
+  frameworksList = await Promise.all(frameworksList.map(async (framework) => {
+    const repoData = await ghRepoData(framework.githubUrl)
+    return {
+      ...framework,
+      starCount: repoData.starCount,
+      languages: repoData.languages.slice(0,2),
+    }
+  }))
+
+  return {
+    props: {
+      ...(await serverSideTranslations(locale!, requiredNamespaces)),
+      frameworksList,
+    },
+  }
+}
+
+const LocalEnvironmentPage = ({ frameworksList }) => {
```

Comment:
Lets declare the types for this page by doing:

```suggestion
const LocalEnvironmentPage = ({ frameworksList }: InferGetStaticPropsType<typeof getStaticProps>)) => {
```